test/e2e/storage/vsphere/vsphere_utils.go

```diff
@@ -44,6 +44,8 @@ import (
 	"k8s.io/kubernetes/pkg/volume/util"
 	"k8s.io/kubernetes/test/e2e/framework"
 	"k8s.io/kubernetes/test/e2e/storage/utils"
+	imageutils "k8s.io/kubernetes/test/utils/image"
+
```

Comment:
nit: remove empty line

Reply:
forgot to actually amend after running hack/update-gofmt.sh  :)